.github/workflows/push.yml

```diff
@@ -40,9 +40,7 @@ jobs:
       run: python3 setup.py install
     - name: Test
       run: |
```

Comment:
Small nit: The command could replace `|` here and it would be all in one line.

Reply:
Used the pattern which is there in the file since before.

Reply:
Jens, Konrad: think we can merge this now

Reply:
Do we actually have to remove it after #151 ? Can you check your other PRs?
I would rather gradually replace bash tests with pytest than lose this coverage.

Reply:
I think after a combination of #162 and #151 we need to remove it anyway

Reply:
OK. Let's close the PR then?

Reply:
Yes, pls

Reply:
I think, this is ready to be merged.

Reply:
@priv-kweihmann final thoughts?

Reply:
lgtm